hub/components/NewWallet/useGovernanceDefaults.ts

```diff
@@ -0,0 +1,136 @@
+import { GovernanceConfig, VoteThresholdType } from '@solana/spl-governance';
+import { PublicKey } from '@solana/web3.js';
+
+import BigNumber from 'bignumber.js';
+
+import BN from 'bn.js';
+import { pipe } from 'fp-ts/lib/function';
+
+import { Rules } from '../EditWalletRules/types';
+import useTreasuryInfo from '@hooks/useTreasuryInfo';
+import { GovernanceTokenType } from '@hub/types/GovernanceTokenType';
+import { GovernanceVoteTipping } from '@hub/types/GovernanceVoteTipping';
+
+import * as RE from '@hub/types/Result';
+
+import { DISABLED_VOTER_WEIGHT } from '@tools/constants';
+
+const configs2defaults = (configs: GovernanceConfig[]) => {
```

Comment:
@SebastianBor this fn takes configs from all wallets in the DAO and then computes what the strictest settings would be for the default config of a new wallet. Obviously this can result in the default configuration to be to disable all voting, but you can't actually do that so whatever. Perhaps you would like to review it.